class BotTaskError(Exception):
    """Исключение"""
    pass
